cmd/server/main.go

```diff
@@ -39,80 +38,64 @@ type Arguments struct {
 
 func mainWithArgs(ctx context.Context, args []string, logger golog.Logger) error {
 	var argsParsed Arguments
+
 	if err := utils.ParseFlags(args, &argsParsed); err != nil {
 		return err
 	}
+
 	if argsParsed.Port == 0 {
+		golog.Global.Debugf("using default port %d ", defaultPort)
```

Comment:
There is a logger being passed to this function. It should be used here instead of directly accessing a global.

Reply:
Done.